packit_service/constants.py

```diff
@@ -207,3 +210,6 @@ def from_number(number: int):
     "*As a project maintainer, you can trigger the build and test jobs manually "
     "via `/packit build` comment or only test job via `/packit test` comment.*",
 )
+
+FAILED_TF_MODEL_STATUS: List[str] = [TestingFarmResult.failed, TestingFarmResult.error]
+FAILED_BUILD_MODEL_STATUS: List[str] = [PG_BUILD_STATUS_FAILURE]
```

Comment:
I meant these to use directly in the code but will leave it up to you (but I can see from the tests it probably causes circular imports).